sdk/src/wallet/operations/transaction/prepare_output.rs

```diff
@@ -285,33 +287,23 @@ where
     ) -> crate::wallet::Result<Address> {
         let transaction_options = transaction_options.into();
 
+        // TODO: more readable than `map_or`?
+        #[allow(clippy::option_if_let_else)]
```

Comment:
could split into
```
let mut remainder_address = None;
if let Some(options) = &transaction_options {
  ...
}
```

Reply:
actually since we re-assign below it should just be one if-let-else

Reply:
Done